actions/errors.go

```diff
@@ -206,13 +206,6 @@ func ErrCode(msg string, err error) error {
 			// for the specified DB security group.
 			rds.ErrCodeAuthorizationAlreadyExistsFault,
 
-			// ErrCodeCustomAvailabilityZoneAlreadyExistsFault for service response error code
-			// "CustomAvailabilityZoneAlreadyExists".
-			//
-			// CustomAvailabilityZoneName is already used by an existing custom Availability
-			// Zone.
-			rds.ErrCodeCustomAvailabilityZoneAlreadyExistsFault,
-
```

Comment:
This constant is no more defined in latest `rds` package. As there is no chance of getting this error from AWS, I have removed this from our file. Please let me know your thoughts.